README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
```

Comment:
Readme File should tell about the project. You should try to explain the project more briefly and describe about the technologies used. Read Microverse Guidelines about the readme [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md). 😊